core/update.php

```diff
@@ -126,6 +127,11 @@ function update_script_selection_form($form, &$form_state) {
   // Warn the user if any updates were incompatible.
   if ($incompatible_updates_exist) {
     backdrop_set_message('Some of the pending updates cannot be applied because their dependencies were not met.', 'warning');
+    foreach ($incompatible_updates as $message_info) {
```

Comment:
Need a check for empty array, because `$incompatible_updates_exist` flag can be set to `TRUE` at 2 different times, see https://github.com/backdrop/backdrop/pull/2017/files#diff-585a9f02ce41902d68f2b9d1c269350fR115